packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -238,6 +241,9 @@ class AndroidSdkVersion implements Comparable<AndroidSdkVersion> {
     if (_canRun(zipalignPath) != null)
       return <String>[_canRun(zipalignPath)];
 
+    if (_canRun(apksignerPath) != null)
+      return <String>[_canRun(apksignerPath) + ' (Requires Android SDK Build Tools 24.0.3 or newer.)'];
```

Comment:
This could tell the user what version of build tools they have, and how to upgrade.